doc/source/data/doc_code/creating_datasets.py

```diff
@@ -590,3 +590,12 @@
 )
 # __read_parquet_az_end__
 # fmt: on
+
+# __read_tfrecords_begin__
+# Create a tabular Dataset by reading a TFRecord file.
+ds = ray.data.read_tfrecords(path)
```

Comment:
Ah one thing I missed, we should add a small (1KB or smaller) TFRecord file to the `python/ray/data/examples` directory and have this example code read from it using our `example://` protocol, similar to the other examples in the "Creating Datasets" feature guide.

This can be done in a follow-up PR if you'd like.

Reply:
Synced offline, going to do this in a follow-up.